cmd/action_config.go

```diff
@@ -14,9 +15,10 @@ import (
 func actionConfig(ctx *cli.Context) error {
 	log := oplog.NewLogger(oplog.AppOut(ctx), oplog.ReadCLIConfig(ctx)).New("role", "mocktimism")
 	oplog.SetGlobalLogHandler(log.GetHandler())
-	cfg, err := config.LoadNewConfig(log, ctx.String(ConfigFlag.Name))
-	if err != nil {
-		log.Error("failed to load config", "err", err)
+	cfg, errs := config.LoadNewConfig(log, ctx.String(ConfigFlag.Name))
+	if len(errs) > 0 {
+		err := errors.Join(errs...)
```

Comment:
you should push `errors.Join` into `config.LoadNewConfig` to keep the `if err != nil` pattern. return a list of errors feels unorthodox

Reply:
Ah yea didn't think about this but I agree

Reply:
Done